.gitignore

```diff
@@ -20,6 +20,8 @@ Makefile.in
 /ABOUT-NLS
 /aclocal.m4
 /autom4te.cache
+/build/
+/build-*/
```

Comment:
Incidentally this is NOT necessary, because a builddir can be named anything (and you don't even match the recommended one which is `builddir/`.

Meanwhile Meson itself always knows which directory you create as a builddir, so it just creates the file `builddir/.gitignore` with the contents `*` for you. :)

Reply:
Maybe not necessary, but it doesn't hurt either. I use build directories for other purposes than meson (e.g. autotools build) in which cases no .gitignore is created.

Reply:
But the purpose of this PR is to migrate from autotools to meson, right? So presumably you don't need that anymore...

Reply:
There is going to be an overlap period similar to other projects migration.

Reply:
You're free to build using meson exclusively but I'm not that far yet and don't want to remove Autotools right now.

I'm also not convinced that meson is the best build system out there (but it's undoubtedly a massive improvement).